lib/WP_Auth0_LoginManager.php

```diff
@@ -537,6 +522,45 @@ protected function query_vars( $key ) {
     return null;
   }
 
+  /**
+   * Get and store state returned from Auth0
+   *
+   * @return object|null
+   */
+  protected function get_state() {
```

Comment:
@lbalmaceda - FYI, I just added this method and it's usage above (using this instead of always parsing the state parameter). It didn't invalidate your review but thought I would point it out.